### PR TITLE
Add interface methods: read user/deal/tag list in data manager

### PR DESCRIPTION
**Pull request by jyx11011**
Add a few interface methods and sub that I think will be useful in various servlets.

### CODE REVIEW

**Review by limli — DISMISSED** [review]

**Review by lisasgoh — DISMISSED** [review]

File: src/main/java/com/google/step/datamanager/UserManagerDatastore.java [hunk 1]
```diff
@@ -118,4 +120,17 @@ public void deleteUser(long id) {
...
+  public List<User> readUsers(List<Long> ids) {
+    List<User> users = new ArrayList<>();
+    for (long id : ids) {
+      try {
+        users.add(readUser(id));
+      } catch (IllegalArgumentException e) {
+        continue;
+      }
+    }
+    return users;
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/step/datamanager/UserManagerDatastore.java]: Is this the final implementation? Can we add tests for this?

File: src/main/java/com/google/step/datamanager/TagManagerDatastore.java [hunk 2]
```diff
@@ -72,4 +74,17 @@ private Tag transformEntityToTag(Entity entity) {
...
+    List<Tag> tags = new ArrayList<>();
+    for (long id : ids) {
+      try {
+        tags.add(readTag(id));
+      } catch (IllegalArgumentException e) {
+        continue;
+      }
+    }
+    return tags;
+  }
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/step/datamanager/TagManagerDatastore.java]: Is this the final implementation? Can we add tests for this?

**Review by SuyashLakhotia — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/step/datamanager/TagManagerDatastore.java (hunk 2 above)
Comment (jyx11011) [reply src/main/java/com/google/step/datamanager/TagManagerDatastore.java]: ok. I refactored read users and read tags according to comments in #57 and add tests.

**Review by SuyashLakhotia — DISMISSED** [review]

**Comment by SuyashLakhotia** [comment]
Please resolve merge conflicts.

**Review by SuyashLakhotia — APPROVED** [review]

**Pull request merged**